src/lib/services/notehub.ts

```diff
@@ -10,6 +10,13 @@ export function isValidDeviceUID(deviceUID: string) {
   return validPrefixes.some((prefix) => deviceUID.startsWith(prefix));
 }
 
+export function isValidProductUID(productUID: string) {
+  const validProductUIDPrefixes = ['com.blues.airnote', 'org.airnote.solar.v1'];
```

Comment:
Per our conversation on Slack, I feel like `product:org.airnote.solar.air.v1` should be considered a valid ProductUID.

Reply:
Good catch. I'll add that as another valid option